setup.py

```diff
@@ -184,6 +184,8 @@ def get_model_zoo_configs() -> List[str]:
         "omegaconf>=2.1",
         "hydra-core>=1.1",
         "black==22.3.0",
+        # Pin protobuf for ONNX/Caffe2 export (>=4.x breaks BC)
+        "protobuf<4.0",
```

Comment:
Latest tests on main branch pass. It seems latest protobuf might have fixed the issue. Please remove this change.

Even if it's not fixed, setting an environment var like https://github.com/facebookresearch/detectron2/pull/4293#issuecomment-1145620185 is a better fix than adding a dependency requirement.